Assets/Scripts/Trigger/buttonTrigger.cs

```diff
@@ -42,5 +42,20 @@ public class buttonTrigger : Trigger {
 		}
 	}
 
+	public override void interact()
+	{
+		UITrigger secretItem = GetComponent<UITrigger> ();
+		if (secretItem != null && !secretItem.found) {
+			secretItem.interact ();
+			buttonPressed = secretItem.replacementSprites [0];
```

Comment:
Instead of putting this in the general buttonTrigger code, could we make another script that extends buttonTrigger and put the logic there, as this is like a one time thing.